drivers/usb/device/usb_dc_stm32.c

```diff
@@ -160,10 +160,6 @@ LOG_MODULE_REGISTER(usb_dc_stm32);
 #define EP0_IN (EP0_IDX | USB_EP_DIR_IN)
 #define EP0_OUT (EP0_IDX | USB_EP_DIR_OUT)
```

Comment:
I am wondering if we should take this:
```
#define USB_CONTROL_OUT_EP0         0
#define USB_CONTROL_IN_EP0          0x80
```
from `./subsys/usb/usb_device.c:104`. Put it into `./include/drivers/usb/usb_dc.h`.
And those two could be removed and unified as well.

This however should be a part of #20236. I will post the PR in free time after this is merged.
Let me know what You think. Just an info.

Reply:
There are more thinks like that, but not relevant to this PR. I will not add new patches here or change the logic. This was tested on all platforms except DW.